README.md

```diff
@@ -1,4 +1,20 @@
 arma_stat_track
 ===============
 
-ARMA stats tracker
+ARMA stats tracker client project.
+This repository handles the game server side of the stats tracking by providing the in-game addon for gathering game related data and the library necessary to provision the data to a backend.
+
+Building
+------
+The library is written in C++ and depends on BOOST.org. It has been written using variants of GCC and as such may need some slight modifications for MSVS or other compiler brands.
```

Comment:
Put each sentence per paragraph on separate lines. Lines without empty row between them are concatenated in output. This decreases git diff when changed.